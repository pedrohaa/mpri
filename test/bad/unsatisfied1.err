Fatal error: exception Failure("Equations do not entail")
