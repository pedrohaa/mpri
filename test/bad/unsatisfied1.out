Esquema: forall a_1_ b_1_. {} -> eq a_1_ b_1_ where a_1_ = b_1_
